src/clj_jargon/permissions.clj

```diff
@@ -292,6 +324,16 @@
            read?   (or write? (= :read permission))]
       (set-permissions cm user fpath read? write? own? recursive?))))
 
+(defn remove-permissions
+  "Remove permissions, recursively where applicable"
+  [cm user fpath]
+  (set-permissions cm user fpath false false false true))
+
+(defn remove-access-permissions
+  "Remove permissions, non-recursively where applicable"
+  [cm user abs-path]
+  (set-permissions cm user abs-path false false false false))
+
```

Comment:
These two functions are reimplementations of those removed lower down in the file, in terms of set-permissions, which does the proper admin behavior. Also, they're shorter.